html/selionGrid.html

```diff
@@ -110,6 +98,26 @@ <h2 id="start-selion-node">Start the Grid Node</h2>
   <div class="container">
     <img src="../images/grid-management-console.jpg" alt="View of the SeLion Grid Management Console">
   </div>
+
+  <h2 id="selion-config">SeLion Grid Configuration File</h2>
+  <p>
+    The SeLion Grid Configuration JSON file allows Grid to load the default properties. File available in
+    <code>$SELION_HOME/config/SeLionConfig.json</code>. Default JSON file will be overrided by <code>-selionConfig</code> argument.
```

Comment:
Should say, `You can specify a different file with the dash option <code>-selionConfig</code>`